src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -1668,4 +1668,32 @@ object SampleWdl {
       """.stripMargin
     override val rawInputs = Map.empty[String, String]
   }
+
+  /**
+    * Outputs which reference themselves.
+    */
+  object InputsAndOutputsAutoReferentiation extends SampleWdl {
```

Comment:
Hmm is there any reason why trying to set `String a` twice shouldn't be a syntax error?